components/pipefy/sources/card-created/card-created.js

```diff
@@ -0,0 +1,22 @@
+const common = require("../common-webhook.js");
+
+module.exports = {
+  ...common,
+  name: "Card Created (Instant)",
+  key: "pipefy-card-created",
+  description: "Emits an event for each new card created in a Pipe.",
+  version: "0.0.1",
+  methods: {
+    ...common.methods,
+    getActions() {
+      return ["card.create"];
+    },
+    getMeta(card, cardData) {
+      return {
+        body: card,
```

Comment:
Question: is `body` supposed to be equal to `cardData` and there was a typo here? Or is `cardData` never actually used?